### PR TITLE
Landing knative kafka broker 1.1 components

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

Updating to 1.1.3 midstream of the kafka broker and sink

Next: Kafka CORE

### CODE REVIEW

File: knative-operator/deploy/resources/knativekafka/controller/1-eventing-kafka-controller.yaml [hunk 1]
```diff
@@ -17,12 +17,36 @@ metadata:
...
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kafka-channel-config
+  namespace: knative-eventing
+  labels:
+    kafka.eventing.knative.dev/release: "devel"
+data:
+  bootstrap.servers: "my-cluster-kafka-bootstrap.kafka:9092"
+
```
Comment (pierDipi) [inline knative-operator/deploy/resources/knativekafka/controller/1-eventing-kafka-controller.yaml:49]: Can we remove this CM?

**Review by pierDipi — CHANGES_REQUESTED** [review]

File: knative-operator/deploy/resources/knativekafka/controller/1-eventing-kafka-controller.yaml (hunk 1 above)
Comment (matzew) [reply knative-operator/deploy/resources/knativekafka/controller/1-eventing-kafka-controller.yaml:49]: Oh, yeah - will do thanks

**Comment by matzew** [comment]
@pierDipi I have removed the `channel` config

**Review by pierDipi — APPROVED** [review]
/lgtm
/approve

:ship:

**Pull request merged**